src/main/scala/cromwell/engine/backend/jes/OperationMetadata.scala

```diff
@@ -14,11 +18,20 @@ object OperationMetadata {
       metadata.get("endTime").map(_.asInstanceOf[String].toDate))
   }
 
-  val Iso8601DateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSXXX")
+  val expectedDateFormat = "yyyy-MM-dd'T'HH:mm:ss.SSSXXX"
+  val defaultDate = DateTime.parse("0")
 
   implicit class EnhancedString(val string: String) extends AnyVal {
-    def toDate: Date = Iso8601DateFormat.parse(string.asInstanceOf[String])
+    def toDate: DateTime = try {
+      val Iso8601DateFormat = DateTimeFormat.forPattern(expectedDateFormat)
```

Comment:
Purely a side comment to @mcovarr but I noticed we already have joda pulled in from some dependency. Not worth changing at this point though.